api/routes.go

```diff
@@ -15,26 +15,22 @@ func (s *server) routes() {
 	// Docker image handlers
 	api.HandleFunc("/images", s.ImageVerificationHandler).Methods(http.MethodHead).Queries("image", "{image}")
 
-	// Service Orchestration handlers
-	api.HandleFunc("/{account}/services", s.ServiceOrchestrationCreateHandler).Methods(http.MethodPost)
-	api.HandleFunc("/{account}/services", s.ServiceOrchestrationUpdateHandler).Methods(http.MethodPut)
-	api.HandleFunc("/{account}/services", s.ServiceOrchestrationDeleteHandler).Methods(http.MethodDelete)
-	api.HandleFunc("/{account}/services/{service}", s.ServiceOrchestrationShowHandler).Methods(http.MethodGet)
+	// Service handlers
+	api.HandleFunc("/{account}/services", s.ServiceCreateHandler).Methods(http.MethodPost)
```

Comment:
This looks a little weird since it's not under `clusters` but I guess there's no other way to do it since it's creating the cluster